glide.yaml

```diff
@@ -27,26 +28,35 @@ import:
   subpackages:
   - github
 - package: github.com/inconshreveable/go-update
+- package: github.com/mitchellh/mapstructure
+  version: db1efb556f84b25a0a13a04aad883943538ad2e0
 - package: github.com/kardianos/osext
 - package: github.com/olekukonko/tablewriter
 - package: github.com/pkg/browser
+# latest viper have issue with IsSet: https://github.com/spf13/viper/issues/276
 - package: github.com/spf13/viper
-  repo: https://github.com/jimmidyson/viper.git
-  vcs: git
+  version: 382f87b929b84ce13e9c8a375a4b217f224e6c65
```

Comment:
it fix the openshift 1.3.1 also now. I also dislike on the commit but sadly this doesn't have any tag/release version :(